### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, lstat '/Users/bienjamin/Dev/features'

### ISSUE THREAD

**Issue by benoct1** [tomkadwill/atom-cucumber-autocomplete#17]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.8.0
**System**: Mac OS X 10.11.5
**Thrown From**: [cucumber-autocomplete](https://github.com/tomkadwill/atom-cucumber-autocomplete) package, v0.6.3
### Stack Trace

Uncaught Error: ENOENT: no such file or directory, lstat '/Users/bienjamin/Dev/features'

```
At ATOM_SHELL_ASAR.js:0

Error: ENOENT: no such file or directory, lstat '/Users/bienjamin/Dev/features'
    at Error (native)
    at fs.lstatSync (fs.js:887:18)
    at Object.fs.lstatSync (ATOM_SHELL_ASAR.js:229:16)
    at Object.scanFeaturesDir (/Users/bienjamin/.atom/packages/cucumber-autocomplete/lib/provider.js:46:13)
    at Object.getCompletions (/Users/bienjamin/.atom/packages/cucumber-autocomplete/lib/provider.js:31:19)
    at Object.getSuggestions (/Users/bienjamin/.atom/packages/cucumber-autocomplete/lib/provider.js:17:17)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:310:56
    at Array.forEach (native)
    at AutocompleteManager.module.exports.AutocompleteManager.getSuggestionsFromProviders (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:289:17)
    at AutocompleteManager.getSuggestionsFromProviders (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:3:61)
    at AutocompleteManager.module.exports.AutocompleteManager.findSuggestions (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:276:19)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:3:61
```
### Commands

```
 46x -4:23.3.0 core:select-right (atom-text-editor.editor.is-focused)
     -3:55.5.0 core:select-left (atom-text-editor.editor.is-focused)
 11x -3:53.2.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -3:34.8.0 core:move-down (atom-text-editor.editor.is-focused)
 30x -3:30.4.0 core:select-left (atom-text-editor.editor.is-focused)
 24x -3:24.1.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -2:51.4.0 core:move-left (atom-text-editor.editor.is-focused)
     -2:49.8.0 core:select-left (atom-text-editor.editor.is-focused)
     -2:49 core:move-left (atom-text-editor.editor.is-focused)
     -2:48.5.0 core:move-right (atom-text-editor.editor.is-focused)
     -2:47.6.0 core:move-left (atom-text-editor.editor.is-focused)
     -2:47.2.0 core:select-right (atom-text-editor.editor.is-focused)
     -2:45.8.0 core:move-down (atom-text-editor.editor.is-focused)
 32x -2:24.2.0 core:backspace (atom-text-editor.editor.is-focused)
 25x -0:47.3.0 core:move-left (atom-text-editor.editor.is-focused)
     -0:24.2.0 editor:newline (atom-text-editor.editor.is-focused)
```
### Config

``` json
{}
```
### Installed Packages

``` coffee
# User
bugzilla-number, v0.1.0
cucumber, v0.5.0
cucumber-autocomplete, v0.6.3
numbers-on-a-pane, v0.1.0

# Dev
No dev packages
```

**Comment by tomkadwill** [comment]
This error occurs because the plugin can't find the features directory. There have been a number of bug reports for this issue so I think it should return a useful error message to the user rather than throwing an exception

**Comment by tomkadwill** [comment]
Hi @benoct1, I've added a fix for this in v0.7.1. Please re-test re-open this issue if the problem still occurs in v0.7.1
